client/my-sites/email/gsuite-add-users/index.jsx

```diff
@@ -286,10 +290,13 @@ class GSuiteAddUsers extends Component {
 						} ) }
 						noticeStatus="is-info"
 					>
-						{ selectedDomainName && (
-							<GoogleMailboxPricingNotice
+						{ selectedDomainName && hasGSuiteWithUs( selectedDomain ) && (
```

Comment:
This function was inside, and it was used two times, but actually the first time that was used was making the component to return null (and display nothing), so the second time that was used was indeed not needed.